README.md

```diff
@@ -1,15 +1,12 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Appointment Teacher Backend
```

Comment:
- [ ] Your app has no tests.  Kindly add some to ensure the integrity and scalability of your app.  
- [ ] Also, you need to please add full documentation for your API.  You can use [rswag](https://github.com/rswag/rswag).  This step will add tests and docs at the same time! 💪